auto_import.sh

```diff
@@ -59,10 +60,10 @@ for dir in $(sudo -u $user find $folder -mindepth 1 -maxdepth 1 -type d -mmin +6
         if [ $modified -gt 1 ]; then
             email=true
         fi
-
+        echo "Will I send an email? $email"
```

Comment:
what is this line for? looks like a debugging line?

Reply:
It is, but it's only saved on our internal cronlogs, not sent to the user. Can still delete it though